src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -710,7 +710,9 @@ Semantics.ConversionKind IConversionExpression.ConversionKind
             }
         }
 
-        bool IConversionExpression.IsExplicit => this.ExplicitCastInCode;
+        public IConversion Conversion => new Conversion(this.ConversionKind, this.SymbolOpt, isExtensionMethod: false);
```

Comment:
I am not sure this is going to produce correct result for all possible situations we need to handle, at least code in SemanticModel looks more complex than this. We should carefully match the way SemanticModel creates Conversion instances for different scenarios and have unit-tests to test them.